schema/schema.json

```diff
@@ -1759,6 +1816,36 @@
       "description": "A unique identifier for a conversation session between a client and agent.\n\nSessions maintain their own context, conversation history, and state,\nallowing multiple independent interactions with the same agent.\n\n# Example\n\n```\nuse agent_client_protocol::SessionId;\nuse std::sync::Arc;\n\nlet session_id = SessionId(Arc::from(\"sess_abc123def456\"));\n```\n\nSee protocol docs: [Session ID](https://agentclientprotocol.com/protocol/session-setup#session-id)",
       "type": "string"
     },
+    "SessionInfo": {
+      "description": "**UNSTABLE**\n\nThis capability is not part of the spec yet, and may be removed or changed at any point.\n\nInformation about a session returned by session/list",
+      "properties": {
+        "_meta": {
+          "description": "Agent-specific metadata (e.g., message count, error status, tags)"
+        },
+        "createdAt": {
+          "description": "ISO 8601 timestamp when session was created",
+          "type": "string"
+        },
+        "cwd": {
+          "description": "Working directory for the session",
+          "type": "string"
+        },
+        "sessionId": {
+          "$ref": "#/$defs/SessionId",
+          "description": "Unique identifier for the session"
+        },
+        "title": {
+          "description": "Human-readable title (may be auto-generated from first prompt)",
```

Comment:
Made a discussion comment [here](https://github.com/orgs/agentclientprotocol/discussions/60#discussioncomment-14936950), but wondering

While true it may be auto-generated, the impact is you shouldn't cache this right? Because it might be initially empty or a fake initial value like "Goose ACP". It seems several flows indeed do a lazy one-time update, but I'm not sure if that's only one time always (e.g. should this just be assumed never cacheable unlike the sessionId, which should me immutable?)